<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vote</class>
 <widget class="QWidget" name="vote">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1425</width>
    <height>1088</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btn_vote">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>940</y>
     <width>231</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Agency FB&quot;;</string>
   </property>
   <property name="text">
    <string>投票</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>391</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 20pt &quot;Agency FB&quot;;</string>
   </property>
   <property name="text">
    <string>候选人列表</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="table">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>401</width>
     <height>681</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>391</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Agency FB&quot;;</string>
   </property>
   <column>
    <property name="text">
     <string>编号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>候选人</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>当前票数</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>150</y>
     <width>291</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;Agency FB&quot;;
color: rgb(170, 0, 255);</string>
   </property>
   <property name="text">
    <string>当前优胜者：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="peo_result">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>260</y>
     <width>351</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 14pt &quot;Agency FB&quot;;</string>
   </property>
   <property name="text">
    <string>排序:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>30</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Agency FB&quot;;</string>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QComboBox" name="box_sort">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Agency FB&quot;;</string>
   </property>
   <item>
    <property name="text">
     <string>请选择</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>按得票数升序</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>按得票数降序</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="box_qurry">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>70</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Agency FB&quot;;</string>
   </property>
   <item>
    <property name="text">
     <string>请选择</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>编号</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>姓名</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>姓名模糊查询</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>显示所有信息</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="qurry_1">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>80</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_back">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>1000</y>
     <width>251</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Agency FB&quot;;</string>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_quit">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>1000</y>
     <width>251</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Agency FB&quot;;</string>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
